client/executor/common/src/sandbox/wasmer_backend.rs

```diff
@@ -100,14 +100,60 @@ pub fn invoke(
 	}
 }
 
+#[cfg(feature = "wasmer-cache")]
+enum CachedModuleErr {
+	FileSystemErr,
+	ModuleLoadErr(FileSystemCache, Hash),
+}
+
+#[cfg(feature = "wasmer-cache")]
+use CachedModuleErr::*;
+
+#[cfg(feature = "wasmer-cache")]
+fn get_cached_module(
+	wasm: &[u8],
+	store: &wasmer::Store,
+) -> core::result::Result<Module, CachedModuleErr> {
+	let fs_cache =
+		FileSystemCache::new("/tmp/substrate-wasmer-cache").map_err(|_| FileSystemErr)?;
```

Comment:
maybe use something like `std::env::temp_dir()`? https://doc.rust-lang.org/std/env/fn.temp_dir.html

Reply:
We may wanna to reuse compiled cache even after node is stopped

Reply:
Done